website/content/en/preview/getting-started/cloudformation.yaml

```diff
@@ -57,4 +57,7 @@ Resources:
               - ec2:DescribeInstanceTypes
               - ec2:DescribeInstanceTypeOfferings
               - ec2:DescribeAvailabilityZones
+          - Effect: Allow
+            Resource: "arn:aws:ssm:*:*:parameter/aws/service/*"
```

Comment:
Lets use the `*` wildcard instead of aws.

Reply:
let me see if CloudFormation has a helper function thats similar to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/partition (I would think/hope so but not a CF person)

Reply:
bingo https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html#cfn-pseudo-param-partition